frontend.py

```diff
@@ -81,8 +79,20 @@ def draw_board(state):
 
     state: State object containing game board and robots
     """
+    window = pyglet.window.Window(state.sizes[0] * TILE_WIDTH,
+                                  state.sizes[1] * TILE_HEIGHT, resizable=True)
```

Comment:
Tady vytváříš to samé okno jako funkce `init_window`. Proto se objeví dvě okna. Pokud `window` potřebuješ, předej ho jako argument funkce.